lms/static/sass/discussion/elements/_actions.scss

```diff
@@ -160,7 +160,7 @@
     }
 
     &.action-vote {
-
+       display: inline-block !important;
```

Comment:
oof, is there a way around using `!important`. I think it's generally frowned upon